community/ndp/pom.xml

```diff
@@ -14,9 +14,11 @@
     <license-text.header>GPL-3-header.txt</license-text.header>
     <licensing.prepend.text>notice-gpl-prefix.txt</licensing.prepend.text>
     <componentversion.skip>true</componentversion.skip>
+    <ascii.doctor.version>1.5.2</ascii.doctor.version>
+    <jsoup.version>1.8.1</jsoup.version>
   </properties>
 
-  <groupId>org.neo4j.build</groupId>
+  <groupId>org.neo4j.ndp</groupId>
```

Comment:
The NDP parent is just a build artifact, similar to the 'community' pom, not something one can depend on. In other places where we've had these build-only poms we've kept them in the 'build' space. Is there a case for moving away from that here?